### PR TITLE
throw exception with warning

### PR DESCRIPTION
**Pull request by olehedza**

### CODE REVIEW

File: src/main/java/core/basesyntax/SqrtOperation.java [hunk 1]
```diff
@@ -8,6 +8,9 @@
  */
 public class SqrtOperation {
     public double sqrt(double number) {
-        return 0.0;
+        if (number < 0) {
+            throw new IllegalArgumentException("Expected non-negative number, got ?");
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/SqrtOperation.java]:
```suggestion
            throw new IllegalArgumentException("Expected non-negative number, got " + number);
```

**Comment by olehedza** [comment]
Done

**Comment by olehedza** [comment]
Done!

**Comment by olehedza** [comment]
Sorry for three commits. Got some troubles)

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**